library_generation/DEVELOPMENT.md

```diff
@@ -0,0 +1,101 @@
+# Linting
+
+When contributing, ensure your changes to python code have a valid
+format.
+
+```
+python -m pip install black
+black library_generation
+```
+
+# Running the integration tests
+
+The integration tests build the docker image declared in
+`.cloudbuild/library_generation/library_generation.Dockerfile`, pull GAPIC
+repositories, generate the libraries and compares the results with the source
+code declared in a "golden branch" of the repo.
+
+It requires docker and python 3.x to be installed.
+
+```
+cd library_generation
+python -m pip install .
+python -m pip install -r requirements.txt
```

Comment:
`python -m pip install library_generation`?

Reply:
So line 21 `cd`s into the `library_generation` folder, so in that case a `pip install .` will pick up the corresponding `setup.py`. Not sure if this is misleading, though. WDYT?

Reply:
As per https://github.com/googleapis/sdk-platform-java/pull/2615#discussion_r1580180271
I added a disclaimer saying that every command will assume the reader's CWD is `library_generation` (except for the docker image instructions)